Source/ARTURLSession.h

```diff
@@ -0,0 +1,27 @@
+//
+//  ARTURLSession.h
+//  Ably
+//
+//  Copyright © 2019 Ably. All rights reserved.
+//
+
+#ifndef ARTURLSession_h
+#define ARTURLSession_h
```

Comment:
❔

Reply:
Xcode introduces this automatically.

Its purpose is to avoid re-compiling the same code more than once if the header gets included more than once; the second time, `ARTURLSession_h` will be defined, so `#ifndef ARTURLSession_h` will be false, so the preprocessor will go directly to the `#endif` at the bottom.

Reply:
I was just pointing out that we don't need this for Objective-C with `#import` / `@import`... being a boilerplate pattern required only for pure C headers 'imported' inline via old skool `#include` statements.